src/entity/user/User.ts

```diff
@@ -183,7 +183,7 @@ type AddNewUserResult =
   | { status: 'failed'; reason: UserFailErrorKeys; error?: Error };
 
 const checkRequiredFields = (data: AddUserData): boolean => {
-  return !!(data && data.id);
+  return !!(data && data.id && data.experienceLevel);
```

Comment:
I think this will break actually for the social signups as they don't send this field in the create endpoint 😭

Reply:
😢